.github/workflows/php.yml

```
@@ -14,11 +14,19 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '7.4'
+
     - name: Validate composer.json and composer.lock
       run: composer validate --strict
 
     - name: Install dependencies
-      run: composer update --prefer-dist --no-progress
+      run: composer install --prefer-dist --no-progress
```

```suggestion
      run: composer update --prefer-dist --no-progress
```

We do not have the lock file in the repo: https://github.com/codelicia/xulieta-json

---

The fact that we do not have the lock file on the repo means that we don't need to use `update` right?

---

Let Composer answer: https://getcomposer.org/doc/01-basic-usage.md#updating-dependencies-to-their-latest-versions

---

Awesome @szepeviktor!